### ISSUE TITLE
Bug with current github files

### ISSUE THREAD

**Issue by Serpens66** [LoneGazebo/Community-Patch-DLL#6496]
_1. Mod version (i.e Date - 3/15):_
17 April 2020

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
building current files yields an error in CvBuilderTactialAI. I replaced m_currentPlot with pPlot, not sure if this is correct.

But main Problem:
   After building the dll, I'm clueless how to use it to start CP?! The official download is not up to date with github I guess (makes sense), but the "Community Patch" folder from github does not include a modinfo file, so how to load it ingame?! I copy pasted the modinfo from offiical release into, but when trying to load the mod, the game simply quits and restarts (without possibility to set a break in dll debugger).

_4. Steps to reproduce (optional):_
Try to build the current files on github.
The missing modinfo thing is probalby only a missunderstanding, so what am I supposed to do, to load my new dll build?

**Comment by ilteroi** [comment]
do you mean CvBuilderTaskingAI? which branch?

anyway it would be good to reference a commit id, but you can just as well use an older revision.

compile it and copy the dll and the pdb to your Mods/communitypatch folder.

**Comment by RecursiveStar** [comment]
It's a merge error, G already said he would fix.

**Comment by Serpens66** [comment]
> 
> 
> do you mean CvBuilderTaskingAI? which branch?
> 
> anyway it would be good to reference a commit id, but you can just as well use an older revision.
> 
> compile it and copy the dll and the pdb to your Mods/communitypatch folder.

yes, sry.
I'm not quite familar with github... no clue how commits/pull requests or such work, it looks very complicated.  I also would like to add a new GameEvent for visibilitycountchange (I made a lua mod to share vision with non-team players, which needs such an event), but I don't know how to commit/pull. I could download the complete reop, make the change and make a push request (or is it pull? isnt that the opposite? strange english).

Anyway, back to the important part:
The problem is, where do I get a working community patch folder? As I saied the official release might be outdated (I got a direct crash to desktop, with debugger showed me that something with monopoly tables is wrong, so I assumed the sql/xml files of that oficiial release are just incomplete, thus I'm asking what to do)
